draft-sharabayko-mops-srt.md

```diff
@@ -1226,7 +1226,11 @@ ACKACK control packets do not contain Control Information Field (CIF).
 
 A Message Drop Request control packet is sent by the sender to the receiver
 when it requests the retransmission of an unacknowledged packet (all or part
-of a message) which is not present in the sender's buffer (e.g. due to an internal error).
+of a message) which is not present in the sender's buffer. This may happen
+in the file/message mode when the message TTL parameter in `srt_sendmsg`
+was a defined timeout and the message was given up retransmission after
+this time, so the packet was removed from the sender buffer.
```

Comment:
```suggestion
of a message) which is not present in the sender's buffer. This may happen, for 
example, when a TTL parameter triggers a timeout, causing a message/packet 
to be removed from the sender's buffer.
```

Reply:
> causing a message/packet to be removed 

I think the method of dropping a message or packet will be different. I guess dropping multiple packets repeatedly is the way to drop a single message.

IMHO, If the deletion of "message" is needed here, it should also be presented how to achieve it. Or, simply, it can be "packet".

Reply:
Ok, I made it a little bit more precise; @stevomatthews please re-review.